pro/cypress/e2e/navigation.cy.ts

```diff
@@ -17,8 +17,10 @@ describe('Navigation', () => {
     logAndGoToPage(login, '/accueil')
 
     cy.stepLog({ message: 'I scroll to my venue' })
-    cy.findByText('Votre page partenaire').scrollIntoView().should('be.visible')
-    cy.findByText('Vos adresses').scrollIntoView().should('be.visible')
+    cy.findByText('Votre page partenaire').scrollIntoView()
```

Comment:
Est ce qu'utiliser un variable intermediaire serait pas mieux ?
`
const addresses = cy.findByText('Vos adresses')
addresses.scrollIntoView()
addresses.should('be.visible')
`
Quelque chose comme ça?

Reply:
on peut faire ça

```
cy.findByText('Vos adresses').as('addresses')
cy.get('@addresses').scrollIntoView()
cy.get('@addresses').should('be.visible')
```

ou ça

```
cy.findByText('Vos adresses').then((elt: any) => {
    cy.wrap(elt).scrollIntoView()
    cy.wrap(elt).should('be.visible')
})
```

mais pas sur que ce soit plus clair